### ISSUE TITLE
Search does not open corresponding articles if not in the currently open zim file

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix-android#708]
Scenario: I have the "mathematics" zim file open and am looking for a medicine-related term (e.g. diacetyl or fexinidazole) which should be found in the Chemistry of Wikimed zim files. The search console does identify them when I type, but when pressing the result it does not take me to the article and simply closes the search window to bring me back where I was.

If the maths zim is open and I search for a maths article, it opens fine.

**Comment by mhutti1** [comment]
Multizim is not yet ready in 2.4 @kelson42 do you want this in 2.4?

**Comment by kelson42** [comment]
No, but if this kind of bug is inside, please mask the option in settings.

**Comment by nemobis** [comment]
Is this a duplicate of https://github.com/kiwix/kiwix-android/issues/54 ?

**Comment by mhutti1** [comment]
Yeah
